README.md

```diff
@@ -120,19 +121,52 @@ You have two options to install the verifier.
 
 #### Option 1: Install via go
 
-```
+If you want to install the verifier, you can run the following command:
+```bash
 $ go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.1
 $ slsa-verifier <options>
 ```
 
-#### Option 2: Compile manually
+Tools like [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates) or [renovate](https://github.com/renovatebot/renovate) use your project's go.mod to identify the version of your dependencies. 
+If you install the verifier in CI, we strongly recommend you follow the steps below to keep the verifier up-to-date:
+
+1. Create a tooling/slsa-verifier.go file containing the following:
+```go
+//go:build tools
+// +build tools
+
+package main
 
+import (
+	_ "github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier"
+)
 ```
+
+1. Run the following commands. (It will create a go.sum file.)
+```bash
+$ go mod init <your-project-name>-tooling
+$ go mod tidy
+```
+
+1. Commit the tooling folder (containing the 3 files slsa-verifier.go, go.mod and go.sum) to the repository.
+1. To install the verifier in your CI, run the following commands:
+```bash
+$ cd tooling
+$ go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier
```

Comment:
FWIW, `go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier` can be run from anywhere. I think you want to do something like `go install .` which will install all of the tooling listed in your `tools.go` or whatever.

Reply:
I tried `go install .` and it did not work: `import "github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier" is a program, not an importable package`. I've replaced the command with `grep _ tooling_test.go | cut -f2 -d '"' | xargs -n1 -t go install`. ptal

Reply:
will merge for the RC, @ianlewis if you have feedback let me know

Reply:
I kind of thought there was a more convenient way of installing them but maybe I'm misremembering. It's been a while since I've seen/used this pattern.